.gitlab/lassen-jobs.yml

```diff
@@ -5,10 +5,65 @@
 # SPDX-License-Identifier: (BSD-3-Clause)
 ##############################################################################
 
-####
-# All lassen build jobs
-nvcc_10_clang_8_0_1 (build and test on lassen):
+##########
+# CPU ONLY
+##########
+
+ibm_clang_9:
+  variables:
+    SPEC: " %clang@9.0.0ibm"
```

Comment:
For my knowledge, is there a reason for the whitespace between there `" %`? I don't see it in some other spec values.

Reply:
I didn't notice that! I don't think it doesn't anything bad, but just in case, I took out any extra space like that in the job specs. Thanks!